ext/pspell/pspell.stub.php

```diff
@@ -4,38 +4,45 @@
 
 function pspell_new(string $language, string $spelling = "", string $jargon = "", string $encoding = "", int $mode = 0): int|false {}
 
-function pspell_new_personal(string $personal, string $language, string $spelling = "", string $jargon = "", string $encoding = "", int $mode = 0): int|false {}
+function pspell_new_personal(
+    string $personal_filename,
```

Comment:
```suggestion
    string $filename,
```
Sufficient imho in this context.

Reply:
Agreed, I didn't realize the function name includes it.